cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -124,11 +125,17 @@ def children_records(self, key, value):
 def access_urls(self, key, value):
     """Translates access urls field."""
     _access_urls = self.get("access_urls", [])
-
+    access_type_mapped = []
+    access_type_list = list(map(int, clean_val("x", value, str)))
+    for i in access_type_list:
+        access_type = mapping(ACCESS_TYPE, str(i))
```

Comment:
`mapping` has `raise_exception` parameter which you could use instead of this `if` statement